vaadin-spreadsheet-flow-parent/vaadin-spreadsheet-flow/src/main/java/com/vaadin/flow/component/spreadsheet/SpreadsheetFilterTable.java

```diff
@@ -210,6 +211,8 @@ protected Button createClearButton() {
         final Button button = new Button("Clear filters");
         button.setDisableOnClick(true);
         button.setEnabled(false);
+        button.addThemeVariants(ButtonVariant.LUMO_TERTIARY,
+                ButtonVariant.LUMO_SMALL);
```

Comment:
I guess since the button has been a standard V10+ button from the start, it doesn't really matter that much, especially as Spreadsheet doesn't really support Material or NoTheme anyway?

Reply:
@rolfsmeds Yes, the whole thing is server side Java.